axelor-business-project/src/main/resources/views/Project.xml

```diff
@@ -715,4 +722,22 @@
       method="computeProjectTotals"/>
   </action-method>
 
+  <action-group name="action-business-project-group-finish-project">
+    <action name="action-business-project-method-check-project-state"
+      if="__config__.app.isApp('business-project') &amp;&amp; isBusinessProject"/>
+    <action name="action-project-group-finish-project"/>
+  </action-group>
+
+  <action-group name="action-business-project-group-project-status-on-change">
+    <action name="action-business-project-method-check-project-state"
+      if="__config__.app.isApp('business-project') &amp;&amp; isBusinessProject &amp;&amp; projectStatus.id == __config__.app.getApp('project')?.completedProjectStatus.id"/>
+    <action name="action-project-group-finish-project"
+      if="__config__.app.isApp('business-project') &amp;&amp; isBusinessProject &amp;&amp; projectStatus.id == __config__.app.getApp('project')?.completedProjectStatus.id"/>
+  </action-group>
```

Comment:
if I'm not mistaken, the action action-business-project-method-check-project-state is called twice, is this normal ?